subprojects/docs/src/docs/userguide/application_plugin.adoc

```diff
@@ -138,8 +138,12 @@ _Depends on_: `jar`, `startScripts`
 +
 Creates a full distribution TAR archive including runtime libraries and OS specific scripts.
 
+[[sec:application_extension]]
+== Application extension
+
+The application plugin adds an extension to the project, which you can use to configure its behaviour. See the link:{groovyDslPath}/org.gradle.api.plugins.JavaApplication.html[JavaApplication] DSL documentation.
```

Comment:
As a British person, I do of course prefer British spelling, but we use US English for the Gradle documentation :smile: "behaviour" -> "behavior"

Reply:
Also,

> See the `JavaApplication` DSL documentation for more information on the properties available on the extension.